src/server/database/migrations/20200720162114-add-secret-fields.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
do questions and festivals actually need secrets?

Reply:
I understood the secrets as a way to regenerate the same hash, to proof that the chainId actually belongs to a (partly) human readable string. So even if the `chainId` is not secret in all cases, you still might want to be able to regenerate it?

Reply:
Well, as long as we're ok with the extra id-swapping in vote validation, I can't think of other complications it would add. Fine with keeping it, might be easier to reason about if all the chainIds work the same way